doc/classes/Physics2DDirectSpaceState.xml

```diff
@@ -38,15 +38,14 @@
 			<argument index="0" name="shape" type="Physics2DShapeQueryParameters">
 			</argument>
 			<description>
-				Check the intersections of a shape, given through a [Physics2DShapeQueryParameters] object, against the space. If it collides with more than a shape, the nearest one is selected. The returned object is a dictionary containing the following fields:
-				pointo: Place where the shapes intersect.
-				normal: Normal of the object at the point where the shapes intersect.
-				shape: Shape index within the object against which the shape intersected.
-				metadata: Metadata of the shape against which the shape intersected. This metadata is different from [method Object.get_meta], and is set with [method Physics2DServer.shape_set_data].
-				collider_id: Id of the object against which the shape intersected.
-				collider: Object against which the shape intersected.
-				rid: [RID] of the object against which the shape intersected.
-				linear_velocity: The movement vector of the object the shape intersected, if it was a body. If it was an area, it is (0,0).
+				Check the intersections of a shape, given through a [Physics2DShapeQueryParameters] object, against the space. If it collides with more than a shape, the nearest one is selected. Note that this method does not take into account the [code]motion[/code] property of the object. The returned object is a dictionary containing the following fields:
+				[code]collider_id[/code]: ID of the object against which the shape intersected.
+				[code]linear_velocity[/code]: The movement vector of the object the shape intersected, if it was a body. If it was an area, it is (0,0).
```

Comment:
Probably useful to mention the value's type here:

The colliding object's velocity [Vector2]. If the object is an [Area2D], the result is [code](0, 0)[/code].